client/state/ui/test/reducer.js

```diff
@@ -91,4 +100,30 @@ describe( 'reducer', () => {
 			expect( state ).to.be.true;
 		} );
 	} );
+
+	describe( '#getSidebarIsCollapsed()', () => {
+		test( 'should default to false', () => {
+			const state = sidebarIsCollapsed( undefined, {} );
+
+			expect( state ).to.be.false;
+		} );
```

Comment:
I'd be tempted to create some initial state in all the tests and `deepFreeze` it so you know your reducers aren't mutating state.

Reply:
Done https://github.com/Automattic/wp-calypso/pull/46045/commits/86a3dc8c4f5dc220f516fbaaf897d04294641592